nautobot/utilities/filters.py

```diff
@@ -540,7 +541,7 @@ class BaseFilterSet(django_filters.FilterSet):
             models.TimeField: {"filter_class": MultiValueTimeFilter},
             models.URLField: {"filter_class": MultiValueCharFilter},
             models.UUIDField: {"filter_class": MultiValueUUIDFilter},
-            MACAddressField: {"filter_class": MultiValueMACAddressFilter},
+            MACAddressCharField: {"filter_class": MultiValueMACAddressFilter},
```

Comment:
Should we not leave the `MACAddressField` filter_class assignment here on the off-chance anyone is using that field type? This seems to just be assigning default filter types. We can leave in `#TODO 2.0` to rip out later.

Reply:
No, this was outright wrong. The `MACAddressField` here was `dcim.forms.MACAddressField` (*form* field class), not `dcim.fields.MACAddressField` (*model* field class), so it wasn't actually doing anything useful since this dict is for mapping *model* fields to filter classes. It was basically a meaningless entry that never would get used.

I could *add* an additional entry for dcim.fields.MACAddressField, but I don't think that's necessary personally.